tools/WinObjC.Logging/WinObjC.Logging.nuproj

```diff
@@ -34,6 +34,12 @@
     <OutputPath>$(SolutionDir)\OutputPackages\$(Configuration)\</OutputPath>
     <ProjectGuid>c47251fe-47c5-464b-8d05-f293da59f272</ProjectGuid>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|x86'" />
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|x86'" />
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|ARM'" />
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|ARM'" />
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|AnyCPU'" />
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'" />
```

Comment:
I'm not touching these. I don't think the Label makes a difference but these are required to have VS not rewrite the .sln out from under us. 

---
In reply to: [102304769](https://github.com/Microsoft/WinObjC/pull/1971#discussion_r102304769) [](ancestors = 102304769)